openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -20,10 +21,14 @@
 from edxnotes.helpers import is_feature_enabled
 from lms.djangoapps.course_api.api import course_detail
 from lms.djangoapps.courseware.access import has_access
+from lms.djangoapps.courseware.access_response import (
+    CoursewareMicrofrontendDisabledAccessError,
```

Comment:
I added a trailing comma here to scratch my own itch :)

I also wrapped some long lines and made a few other cosmetic changes :P